tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -1091,12 +1092,14 @@ TEST_F(EnergyPlusFixture, ChillerIPLVTestAirCooled)
                     state->dataChillerElectricEIR->ElectricEIRChiller(1).ChillerEIRFTIndex,
                     state->dataChillerElectricEIR->ElectricEIRChiller(1).ChillerEIRFPLRIndex,
                     state->dataChillerElectricEIR->ElectricEIRChiller(1).MinUnloadRat,
-                    IPLV,
+                    IPLVSI,
+                    IPLVIP,
                     ObjexxFCL::Optional<const Real64>(),
                     ObjexxFCL::Optional_int_const(),
                     ObjexxFCL::Optional<const Real64>());
 
-    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 3.87); // 13.20 IPLV
+    EXPECT_DOUBLE_EQ(round(IPLVSI * 100) / 100, 3.88); // 3.88 IPLV.SI (AHRI Std 551/591 Test Condition)
+    EXPECT_DOUBLE_EQ(round(IPLVIP * 100) / 100, 3.89); // 13.27 IPLV.IP (AHRI Std 550/590 Test Condition)
```

Comment:
Modified existing unit test to test/check the `IPLV.SI` and `IPLV.IP`. Not that `IPLV.SI` is determined using the AHRI Std 551/591 test conditions and the `IPLV.IP` is determined using the AHRI Std 550/590 test conditions and the units for both variables in this test are `W/W`.